### ISSUE TITLE
Support for dual sim

### ISSUE THREAD

**Issue by melvinchng** [react-native-webrtc/react-native-carrier-info#14]
When the following method is used on a dual sim device with two active sim cards inserted, the method below returns the second sim's MCC + MNC. When the second sim is removed, the first sim's MCC + MNC is reported correctly. I am writing to check if it is possible to request a feature to get both sims' results.

```
CarrierInfo.mobileNetworkOperator()
.then((result) => {
  Alert.alert('MCC + MNC', result);
});
```

**Comment by shrikantr-iprogrammer** [comment]
Yes, support for dual Sim is very important these days. Almost everyone is using 2 Sims. iPhone also started support for 2 Sims.
